promptflow/data/web_classification/asset.yaml

```diff
@@ -0,0 +1,6 @@
+name: web-classification                 
```

Comment:
I think it is a good idea. But do we have a display name without prefix? When customer use prompt flow, they can select the display name in drop down.

Reply:
Doesn't look like it: https://learn.microsoft.com/en-us/azure/machine-learning/reference-yaml-data?view=azureml-api-2

Reply:
as dataset do not have display name, I suggest we do not use prefix. It will look redundant when customer select data in drop down: 

- promptflow-data1
- promptflow-data2
- promptflow-data3

Reply:
Will there be any other prompt flow related data assets? I'm only suggesting the prefix if it would be useful for display purposes, or help the customer find them.

Reply:
We have around 7 samples by //build, each sample has a corresponding dataset (some samples can share the same data). What we can do to help customer find them, is to set a default value in the drop down list when the flow is created from samples.